### DISCUSSION TITLE
Docusaurus: project: A CHANGELOG.md success story

### DISCUSSION THREAD

**Discussion by tony** [facebook/docusaurus#6618] [General] [answered]
More and more projects have been migrating away from CHANGES files in favor of GitHub Releases.

Docusaurus' [`CHANGELOG.md`](https://github.com/facebook/docusaurus/blob/main/CHANGELOG.md):
- [x] One file
- [x] Markdown
- [x] Sync'd with git tags
 
  Example: [v2.0.0-beta.13](https://github.com/facebook/docusaurus/blob/v2.0.0-beta.13/CHANGELOG.md)
- [x] Easily linkable to headers

  [Linking to v2.0.0-beta10](https://github.com/facebook/docusaurus/blob/v2.0.0-beta.15/CHANGELOG.md#200-beta10-2021-12-09)
- [x] Includes date

  Bonus, date shows in headers: `CHANGELOG.md#200-beta10-2021-12-09`

GitHub releases: 
- There's no way to view it all on one page and Cmd/Ctrl-F
- Releases are spread across 10, 20, 30, 40 + pages

  ![image](https://user-images.githubusercontent.com/26336/152665670-80d1b477-5412-4e0a-9716-73f456c64291.png)

   Above: webpack's releases

**Comment by Josh-Cena** [comment, marked as answer]
We do both. I do find value in GitHub releases (being able to share a particular version's changes; good for marketing) as well as the CHANGELOG (a centralized place to browse all versions). In addition, we now have this: https://docusaurus.io/changelog
